src/Blockcore.Indexer.Core/Models/AddressBalanceResult.cs

```diff
@@ -0,0 +1,8 @@
+namespace Blockcore.Indexer.Core.Models
+{
+   public class AddressBalanceResult
```

Comment:
Why not return a list of `QueryAddress` as already fetch that data from disk (apart for mempool)